crm_phonecall_summary_restrict/models/crm_phonecall.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from psycopg2 import IntegrityError
+from openerp import api, fields, models, SUPERUSER_ID
+
+
+class CRMPhonecall(models.Model):
+    _inherit = "crm.phonecall"
+
+    name = fields.Char(
+        related="summary_id.name",
+        store=True,
+        required=False,
+        readonly=True)
+    summary_id = fields.Many2one(
+        "crm.phonecall.summary",
+        "Summary",
+        required=True,
+        ondelete="restrict")
+
+    def _set_default_value_on_column(self, cr, column_name, context=None):
```

Comment:
This one gets called only at install time. Fixes https://github.com/OCA/crm/pull/78#issuecomment-188317996.